src/utils/node_byte_utils.ts

```diff
@@ -6,7 +6,7 @@ type NodeJsEncoding = 'base64' | 'hex' | 'utf8' | 'binary';
 type NodeJsBuffer = ArrayBufferView &
   Uint8Array & {
     write(string: string, offset: number, length: undefined, encoding: 'utf8'): number;
-    copy(target: Uint8Array, targetStart: number, sourceStart: number, sourceEnd: number): number;
+    copy(target: Uint8Array, targetStart: number, sourceStart?: number, sourceEnd?: number): number;
     toString: (this: Uint8Array, encoding: NodeJsEncoding, start?: number, end?: number) => string;
```

Comment:
```suggestion
    copy(target: Uint8Array, targetStart: number, sourceStart: number, sourceEnd: number): number;
```

It appears you do not use this change? I actually do not see any usage of this API 🤔 but for here we can just reduce the diff?

Reply:
Another good catch. I was using copy at some point but `Uint8Array.set` is much faster. Will remove this change